anaSbitReadout

```diff
@@ -0,0 +1,375 @@
+    canv_3x8.SaveAs(filename+'/ChannSummary.png')
+
+    # Uncomment for more png plots
+    """
+    canv_3x8 = make3x8Canvas(
+                             name="SbitSize_3x8canv",
+                             initialContent=vfat_h_sbitSize,
+                             initialDrawOpt="hist",
+                             secondaryContent=None,
+                             secondaryDrawOpt="hist")
+    canv_3x8.SaveAs(filename+'/SbitSizeSummary.png')
+
+    canv_3x8 = make3x8Canvas(
+                             name="L1A_Delay_3x8canv",
+                             initialContent=vfat_h_delay,
+                             initialDrawOpt="hist",
+                             secondaryContent=None,
+                             secondaryDrawOpt="hist")
+    canv_3x8.SaveAs(filename+'/L1A_DelaySummary.png')
+    """
+
+    # saveSummaryByiEta
+    saveSummaryByiEta(ieta_h_strip, name='%s/ietaStripSummary.png' %
+                      filename, trimPt=None, drawOpt="")
+    saveSummaryByiEta(ieta_h_ch, name='%s/ietaChanSummary.png' %
+                      filename, trimPt=None, drawOpt="")
+    saveSummaryByiEta(ieta_h_sbitSize, name='%s/ietaSbitSizeSummary.png' %
+                      filename, trimPt=None, drawOpt="")
+    saveSummaryByiEta(ieta_h_delay, name='%s/ietaDelaySummary.png' %
+                      filename, trimPt=None, drawOpt="")
+
+    # Making&Filling folders in the TFile
+    outT.Write()
+    vfatDir = outF.mkdir("VFATs")
+    ietaDir = outF.mkdir("iETAs")
+    vfatDir.cd()
+    for vfat in range(0, 24):
+        tempDir = vfatDir.mkdir("VFAT%i" % vfat)
+        tempDir.cd()
+        vfat_h_strip[vfat].Write()
+        vfat_h_ch[vfat].Write()
+        vfat_h_delay[vfat].Write()
+        vfat_h_sbitSize[vfat].Write()
+
+    ietaDir.cd()
+    for ieta in range(1, 9):
+        tempDir = ietaDir.mkdir("iETA%i" % ieta)
+        tempDir.cd()
+        ieta_h_strip[ieta].Write()
+        ieta_h_ch[ieta].Write()
+        ieta_h_delay[ieta].Write()
+        ieta_h_sbitSize[ieta].Write()
+
+    # 2D histos aesthetics
+    line1 = r.TLine(128, 0.5, 128, 8.5)
+    line2 = r.TLine(256, 0.5, 256, 8.5)
+    line1.SetLineColor(r.kRed)
+    line1.SetLineWidth(3)
+    line2.SetLineColor(r.kRed)
+    line2.SetLineWidth(3)
+
+    canv = r.TCanvas("summary", "summary", 500*8, 500*3)
+    canv.SetGridy()
+    canv.cd()
+    hh_ieta_strip[0].Draw('9COLZ')
+    line1.Draw()
+    line2.Draw()
+    canv.Update()
+    canv.SaveAs(filename+'/StripvsiEta.png')
+
+    canv.Clear()
+    canv.cd()
+    hh_ieta_ch[0].Draw('COLZ')
+    line1.Draw()
+    line2.Draw()
+    canv.Update()
+    canv.SaveAs(filename+'/ChvsiEta.png')
+
+    outF.Close()
+    print "\n---Took", (time.time() - start_time) / \
+        int(idx), "seconds for each .dat file---"
+    print "\nGaranting permission to ", filename, "..."
+    os.system("chmod -R 770 "+filename)
+    print "Data stored in", filename+'/'+outfilename, "\nBye now"
+    exit(-1)
```

Comment:
Is this a good exit or a bad exit (there are standardized exit codes, so it's best to not try to reinvent the wheel)?
@bdorney enumerated some standard exit codes, and it would probably be good to use those here